dialog/en-us/going.to.sleep.short.dialog

```diff
@@ -0,0 +1 @@
+Okay.
```

Comment:
Maybe add a second line as an alternative with just: `Going to sleep` slightly longer but not by much. (You don't need to translate it to the other languages if that's hard, that will be picked up on translate.mycroft.ai)